api/v6/report_server.thrift

```diff
@@ -383,9 +383,16 @@ service codeCheckerDBAccess {
 
   // remove bug results from the database
   // PERMISSION: PRODUCT_STORE
```

Comment:
Please extend the API description with a Warning about the behavior change in the API call. (Will not remove the run only results will be deleted and use the new api call removeRun to actually delete a run with the results together.